apps/dashboard/pkg/apis/dashboard/v2alpha1/dashboard_spec.cue

```diff
@@ -708,6 +708,10 @@ VariableRefresh: *"never" | "onDashboardLoad" | "onTimeRangeChanged"
 // Accepted values are `dontHide` (show label and value), `hideLabel` (show value only), `hideVariable` (show nothing).
 VariableHide: *"dontHide" | "hideLabel" | "hideVariable"
 
+// Determine the origin of the adhoc variable filter
+// Accepted values are `dashboard` (filter originated from dashboard), or `scopes` (filter originated from scopes).
+FilterOrigin: *null | "dashboards" | "scopes"
```

Comment:
This feels odd- filter origin should be either dashboard(singular) or scopes. The null-ish value is controlled through optional property in the ad hoc filters definition.

Reply:
Yeah, I agree singular is nicer, but I'll apply it for both for consistency. Plural is how the FilterOrigin enum was defined from the beginning in the adhocs